benchmark_config_directory: $(find eurobench_benchmark_server)/config/benchmark_config
benchmark_output_directory: "~/eurobench_output"

robot_names:
  - TestingBot
  - TestRobot2

excluded_topics:
  - /bmserver/state

# Seconds until benchmark start time
benchmark_countdown: 5